scripts/quests/windurst/Creepy_Crawlies.lua

```diff
@@ -0,0 +1,103 @@
+-----------------------------------
+-- Creepy Crawlies
+-----------------------------------
+-- Log ID: 2, Quest ID: 39
+-- Illu Bohjaa : !pos 21.35 -5 125 241
+-----------------------------------
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/zone')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.WINDURST, xi.quest.id.windurst.CREEPY_CRAWLIES)
+
+quest.reward =
+{
+    title = xi.title.CRAWLER_CULLER,
+    fame = 30,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE
+        end,
+
+        [xi.zone.WINDURST_WOODS] =
+        {
+            ['Illu_Bohjaa'] = quest:event(333, 0, xi.items.SPOOL_OF_SILK_THREAD, xi.items.PAPAKA_GRASS, xi.items.CRAWLER_CALCULUS),
+
+            onEventFinish =
+            {
+                [333] = function(player, csid, option, npc)
+                    if option == 1 then
+                        quest:begin(player)
+                    end
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.WINDURST_WOODS] =
+        {
+            ['Illu_Bohjaa'] =
+            {
+                onTrade = function(player, npc, trade)
+                    if
+                        npcUtil.tradeHas(trade, { { xi.items.SPOOL_OF_SILK_THREAD, 3 } }) or
+                        npcUtil.tradeHas(trade, { { xi.items.CRAWLER_CALCULUS, 3 } })
+                    then
+                        return quest:event(335, 600 * xi.settings.main.GIL_RATE, xi.items.SPOOL_OF_SILK_THREAD, 0, xi.items.CRAWLER_CALCULUS)
+                    end
+                end,
+
+                onTrigger = quest:event(334, 0, xi.items.SPOOL_OF_SILK_THREAD, 0, xi.items.CRAWLER_CALCULUS),
+            },
+
+            onEventFinish =
+            {
+                [335] = function(player, csid, option, npc)
+                    if quest:complete(player) then
+                        player:confirmTrade()
+                        player:addGil(600 * xi.settings.main.GIL_RATE) -- moved from reward section due to csis string ID#6549 included message special for gil
```

Comment:
Since there is no difference in repeat, this can just be added to the quest reward table with a value of 600.  Settings gil rate is automatically applied there, and there is no harm for calling quest:complete() after completed.

Reply:
Note: You still need that multiplier in the event, which you have

Reply:
I think the issue I had with putting it in the reward section, the "you received XXX gil" twice, because it is in the CS itself. Thoughts?

Reply:
Oh, I've seen that before, and in this case I agree with your implementation.

Reply:
Might want to leave a comment in that block to indicate that the CS provides the gil reward message